cube/cube.cpp

```diff
@@ -269,79 +273,77 @@ struct Demo {
     void run_display();
 #endif
 
+    std::string name;  // Name to put on the window/icon
```

Comment:
why not initialize this to `APP_SHORT_NAME` here, since all other initialization is happening here?

Reply:
Fixed. I thought I couldn't initialize std::string like that, but I was wrong.